litgpt/utils.py

```diff
@@ -35,6 +35,21 @@ def init_out_dir(out_dir: Path) -> Path:
     return out_dir
 
 
+def find_resume_path(resume: Union[bool, Literal["auto"], Path], out_dir: Path) -> Optional[Path]:
+    if not resume or isinstance(resume, Path):
+        return resume
+
+    ckpts = list(out_dir.rglob("step-*/*.pth"))
```

Comment:
Found on internet:
```text
If you pass max() an empty iterable as an argument, it will throw a ValueError. The arguments to max() cannot be null. To avoid this error, Python max() accepts the default parameter. Even if the list is empty, max() returns the default value.
```

Reply:
That's useful, so I can add None as the default.

Reply:
Thanks!